cpu/cortexm_common/vectors_cortexm.c

```diff
@@ -129,6 +129,9 @@ void reset_handler_default(void)
     }
 
 #ifdef CPU_HAS_BACKUP_RAM
+#if BACKUP_RAM_HAS_INIT
+        backup_ram_init();
```

Comment:
No need for the indent

```suggestion
    backup_ram_init();
```